apps/s_server.c

```diff
@@ -1884,7 +1884,7 @@ int s_server_main(int argc, char *argv[])
         }
         BIO_printf(bio_err, "id_prefix '%s' set.\n", session_id_prefix);
     }
-    SSL_CTX_set_quiet_shutdown(ctx, 1);
+    SSL_CTX_set_quiet_shutdown(ctx, 0);
```

Comment:
Probably this line can just be removed since this is the default anyway.

Reply:
Fixed.